reassembler.go

```diff
@@ -197,12 +198,12 @@ func (e *event) IsExpired() bool {
 // Type - eventList
 
 type eventList struct {
-	sync.Mutex
-	seqs    sequenceNumSlice
 	events  map[sequenceNum]*event
-	lastSeq sequenceNum
+	seqs    sequenceNumSlice
 	maxSize int
 	timeout time.Duration
+	sync.Mutex
```

Comment:
The location of the mutex in the struct has human semantic burden; mutexes are conventionally placed above struct fields they protect in Go idiom.

Reply:
reverted this change.